src/components/Button/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import './Button.css';
+
+const Button = ()=>(
+        <div className="div-btn">
+        </div>
```

Comment:
If it is empty you should not close it. 
It should look like this <div className="" />

Reply:
Also, use appropriate classNames. It is not a good idea to call it like div, span, input, etc.